### PR TITLE
fix(Label): change hover style for label with copy, fix label with single icon

### PR DESCRIPTION
**Pull request by mfyodorov**
- change hover style for label with copy: [Storybook](https://preview.yandexcloud.dev/uikit/351/?path=/story/components-label--interactions)
- fix label with a single icon: [Storybook](https://preview.yandexcloud.dev/uikit/351/?path=/story/components-label--icon)

### CODE REVIEW

File: src/components/Label/Label.scss [hunk 1]
```diff
@@ -22,21 +22,25 @@ $transitionTimingFunction: ease-in-out;
...
+        }
+
+        &_side_left {
             left: 2px;
         }
 
-        &_right {
+        &_side_right {
             right: 0;
         }
```
Comment (mfyodorov) [inline src/components/Label/Label.scss:41]: like in the Button

**Review by amje — APPROVED** [review]

File: src/components/Label/Label.tsx [hunk 2]
```diff
@@ -58,60 +65,62 @@ export const Label = React.forwardRef<HTMLDivElement, LabelProps>(function Label
...
 
-    const content = <div className={b('text')}>{children}</div>;
+    const content = hasContent && <div className={b('text')}>{children}</div>;
 
     const renderCopyButton = (status?: CopyToClipboardStatus) => {
         return (
-            typeCopy && (
-                <div className={b('icon', {right: true, copy: true})}>
+            <Button
+                onClick={onClose}
```
Comment (ValeraS) [inline src/components/Label/Label.tsx:97]: Why `onClose` here for copy button?

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: src/components/Label/Label.tsx (hunk 2 above)
Comment (mfyodorov) [reply src/components/Label/Label.tsx:97]: this's typo, thanks, i'll fix it

File: src/components/Label/Label.tsx [hunk 3]
```diff
@@ -58,14 +60,18 @@ export const Label = React.forwardRef<HTMLDivElement, LabelProps>(function Label
...
     } = props;
 
+    const hasContent =
+        children && Array.isArray(children) ? children.length > 0 : Boolean(children);
+
     const typeDefault = type === 'default';
-    const typeClose = type === 'close';
-    const typeCopy = type === 'copy';
+    const typeClose = type === 'close' && hasContent;
+    const typeCopy = type === 'copy' && hasContent;
```
[the remark this answers is not in the document]
Comment (mfyodorov) [reply src/components/Label/Label.tsx:85]: Exactly. There is some issue to implement that behaviour and it makes no sense as it seems to me.

File: src/components/Label/Label.tsx [hunk 4]
```diff
@@ -151,16 +157,15 @@ export const Label = React.forwardRef<HTMLDivElement, LabelProps>(function Label
         event.stopPropagation();
     };
 
-    const labelWithCopy =
-        typeCopy && copyText && !disabled ? (
+    if (hasCopy && copyText) {
```
Comment (ValeraS) [inline src/components/Label/Label.tsx:160]:
```suggestion
    if (hasCopy) {
```

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: src/components/Label/Label.tsx (hunk 4 above)
Comment (mfyodorov) [reply src/components/Label/Label.tsx:160]: I check copyText value because typescript highlights text prop of the CopyToClipboard below

**Review by ValeraS — APPROVED** [review]

File: src/components/Label/Label.tsx (hunk 4 above)
Comment (mfyodorov) [reply src/components/Label/Label.tsx:160]: I thought about this double check but I'm think it's ok

**Pull request merged**